docs/openapi/components/schemas/common/EntrySummary.yml

```diff
@@ -0,0 +1,396 @@
+    type: string
+    $linkedData:
+      term: importingCarrier
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#carrierParty
+  transportMode:
+    title: Mode Of Transportation
+    description: The Mode of Transportationfrom the available options Air, Ocean, Rail, Truck, Hand Carry, Pipeline, or Other.
+    type: string
+    $linkedData:
+      term: transportMode
+      '@id': https://w3id.org/traceability#transportMode
+  countryOfOrigin:
+    title: Country of Origin
+    description: >-
+      Record the country of origin utilizing the International Organization for Standardization(ISO) country code located in Annex B of the HTS.The country of origin is the country of manufacture, production, or growth of any article.If the article consists of material produced, derived from, or processed in more than oneforeign territory or country, or insular possession of the U.S., it shall be considered aproduct of that foreign territory or country, or insular possession, where it last underwenta substantial transformation. For reporting purposes only on the CBP Form 7501,whenever merchandise has been returned to the U.S. after undergoing repair,alteration, or assembly under HTS heading 9802, the country of origin should be shownas the country in which the repair, alteration, or assembly was performed.When merchandise is invoiced in or exported from a country other than that in which itoriginated, the actual country of origin shall be specified rather than the country ofinvoice or exportation.When an entry summary covers merchandise from more than one country of origin,record the word "MULTI" in this block. In column 27, directly below the line number,prefixed with the letter "O," indicate the ISO code corresponding to each line item.
```

Comment:
```suggestion
      Record the country of origin utilizing the International Organization for Standardization (ISO) country code located in Annex B of the HTS. The country of origin is the country of manufacture, production, or growth of any article. If the article consists of material produced, derived from, or processed in more than one foreign territory or country, or insular possession of the U.S., it shall be considered a product of that foreign territory or country, or insular possession, where it last underwent a substantial transformation. For reporting purposes only on the CBP Form 7501, whenever merchandise has been returned to the U.S. after undergoing repair, alteration, or assembly under HTS heading 9802, the country of origin should be shown as the country in which the repair, alteration, or assembly was performed. When merchandise is invoiced in or exported from a country other than that in which it originated, the actual country of origin shall be specified rather than the country of invoice or exportation. When an entry summary covers merchandise from more than one country of origin, record the word "MULTI" in this block. In column 27, directly below the line number, prefixed with the letter "O," indicate the ISO code corresponding to each line item.
```